client/src/index.jsx

```diff
@@ -31,8 +25,20 @@ const App = () => {
     userLocation();
   }, [])
 
-  const [location, setLocation] = useState({});
+  const [email, setEmail] = useState("");
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [accountType, setAccountType] = useState("");
+  const [preferredIndustry, setPreferredIndustry] = useState("");
+  const [zipCode, setZipCode] = useState("");
+  const [company, setCompany] = useState("");
+  const [coord_lat, setCoord_lat] = useState();
+  const [coord_long, setCoord_long] = useState();
+
+  const [location, setLocation] = useState({}); // ASK nick if still need this.
```

Comment:
If you want to pull the default location without having to log in, then yes I need it. If we don't go with default locations then no.

Reply:
right. I can take the comments out now. Thanks!